packages/web/src/components/collectibles/components/CollectiblesPage.module.css

```diff
@@ -126,6 +126,22 @@
   color: var(--white);
 }
 
+.chain {
+  position: absolute;
+  bottom: var(--unit-3);
+  right: var(--unit-3);
+  height: var(--unit-6);
+  width: var(--unit-6);
+  display: flex;
+  align-items: center;
+  z-index: 10000;
+  background: var(--static-white);
+  border: 1px solid var(--static-neutral-light-7);
+  border-radius: 50%;
+  box-shadow: 0px 0px 1px -2px var(--tile-shadow-1), 0px 1px 0px -2px #e3e3e3,
```

Comment:
can we steal #e3e3e3 from somwehere?

https://github.com/AudiusProject/audius-client/blob/98814ea4d0a4d8b6b9c9332a6d5cd70b282f4422/packages/web/src/utils/theme/default.ts#L54

or maybe just this shadow itself?

Reply:
Yeah def nice catch